source/gwork/include/Gwork/Skins/TexturedBase.h

```diff
@@ -1006,7 +1003,7 @@ namespace Gwk
                 Textures.Input.Slider.H.Normal.DrawCenter( GetRender(), control->GetRenderBounds() );
             }
 
-            void DrawTreeNode( Controls::Base* ctrl, bool bOpen, bool bSelected, int iLabelHeight, int iLabelWidth, int iHalfWay, int iLastBranch, bool bIsRoot )
+            virtual void DrawTreeNode( Controls::Base* ctrl, bool bOpen, bool bSelected, int iLabelHeight, int iLabelWidth, int iHalfWay, int iLastBranch, bool bIsRoot ) override
```

Comment:
Don't worry about it. If you add all the overrides (and clang gives warning help) then I think I can write a regex to get rid of the virtuals.